utils/vaults.json

```diff
@@ -1019,5 +1019,17 @@
     "COINGECKO_SYMBOL": "weth",
     "VAULT_STATUS": "new",
     "CHAIN_ID": 1
+  },
+    "ywxdai": {
+    "TITLE": "Basic Wrapped xDai Farmer",
+    "LOGO": "🏆🚀",
+    "VAULT_ABI": "yVaultV2",
+    "VAULT_TYPE": "experimental",
+    "VAULT_ADDR": "0xE35f0516D330395a3878eB60114349b6325ec02E",
+    "WANT_ADDR": "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d",
+    "WANT_SYMBOL": "wxDai",
+    "COINGECKO_SYMBOL": "wrapped-xdai",
+    "VAULT_STATUS": "withdraw",
```

Comment:
the status should be "new". If now it won't show it unless you have funds in it. There's a script that will create this for you :)